specs/language/introduction.tex

```diff
@@ -273,21 +273,46 @@
 
 \Sec{\acrshort{hlsl} Memory Models}{Intro.Memory}
 
-\p Memory accesses for \gls{sm} 5.0 and earlier operate on 128-bit slots aligned
-on 128-bit boundaries. This optimized for the common case in early shaders where
-data being processed on the GPU was usually 4-element vectors of 32-bit data
-types.
-
-\p On modern hardware memory access restrictions are loosened, and reads of
-32-bit multiples are supported starting with \gls{sm} 5.1 and reads of 16-bit
-multiples are supported with \gls{sm} 6.0. \gls{sm} features are fully
-documented in the \gls{dx} Specifications, and this document will not attempt to
-elaborate further.
+\p The fundamental storage unit in HLSL is a \textit{byte}, which is comprised
+of 8 \textit{bits}. Each \textit{bit} stores a single value 0 or 1. Each byte
+has a unique \textit{memory location}, alternatively called an \textit{address}.
+
+\p Each read or write to a memory location is called a \textit{memory access}.
+Operations that perform memory accesses are called \textit{memory operations}. A
+memory operation may operate on one or more memory locations. A memory operation
+must not alter memory at a location not contained in the set of memory locations it
+is operating on\footnote{Two subtle notes here: (1) A bit-field's memory location
```

Comment:
Defining the commonly understood term `memory access granularity` and specifying the access granularity to be a `byte` along with subsequent usage of the term may be a better option than using "bit-fields".

Reply:
Bit-fields are a specific language structure that has unique properties in the memory model because of their unique packing behavior.